remo/profiles/views.py

```diff
@@ -87,8 +87,14 @@ def profile_date_form_validation(form):
                       'Onboarding': 'onboarding_group',
                       'Newsletter': 'newsletter_group'}
 
+            admin_only_groups = ['Admin', 'Peers']
+
+            user_is_admin = request.user.groups.filter(name='Admin').exists()
             for group_db, group_html in groups.items():
                 if Group.objects.filter(name=group_db).exists():
+                    if group_db in admin_only_groups and not user_is_admin:
```

Comment:
I would suggest to completely hide the groups from users that shouldn't be able to edit them.